source/channels/react-to-messages.rst

```diff
@@ -4,84 +4,74 @@ React to messages using emojis
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-Emojis are small, digital images or icons used to communicate or express concepts such as emotions and physical gestures in your messages. 
+Emojis are small, digital images or icons you can use to communicate or express concepts such as emotions and physical gestures in your messages. 
+
+From Mattermost v6.1 and in Mattermost Cloud, you can react to messages quickly by selecting a recently used emoji. Recent emojis are sorted based on the number of times you've used the emoji. If your most recent reactions aren't visible, you can enable quick reactions by selecting **Settings > Display > Quick reactions on messages**.
 
 .. tabs::
 
   .. tab:: Web browser/desktop app
-      
-    **To react to messages with emoji** 
-        
+              
     Select the **Smile** icon |smile-icon| inside the Mattermost message input box to open the emoji picker. If the existing emojis available in Mattermost don't cover what you want to express, you can also `upload your own custom emoji <#upload-custom-emojis>`__.
 
        .. |smile-icon| image:: ../images/smile-icon.png
            :alt: Smile icon
 
-    .. tip::
+    Hover over a message to access recently used emojis. 
+
+    .. image:: ../images/recent-emojis.png
+      :alt: React to messages quickly by selecting one of your most recently used emojis.
      
-       You can also react to messages by typing ``:`` followed by at least two characters of the word describing the emoji. This opens an emoji autocomplete. Descriptions include skin tone details for people-based emojis, where supported.
+    You can also react to messages by typing ``:`` followed by at least two characters of the word describing the emoji. This opens an emoji autocomplete. Descriptions include skin tone details for people-based emojis, where supported.
 
-        .. image:: ../images/emojiautocomplete.png
-            :alt: Emoji autocomplete
+    .. image:: ../images/emojiautocomplete.png
+      :alt: Emoji autocomplete
         
-    **Select recent emojis**
+  .. tab:: Mobile app
```

Comment:
Smile button is to add emojis in post drafts. Reacting is always a long press then select the recently used emoji, or select the smile button in that menu to add a reaction of a different kind